src/style.css

```diff
@@ -1,3 +1,86 @@
+:root {
+  --color-hover: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
```

Comment:
Consider adding fallback fonts for greater compatibility. If the "Roboto" font fails to load for any reason, the browser will try to use the next font in the list. If no fonts are available, it will resort to a default font. Here's an example of how you could implement this: `font-family: Roboto, Arial, sans-serif;`